src/Elcodi/Component/Core/Generator/RandomGenerator.php

```diff
@@ -31,7 +31,7 @@ class RandomGenerator implements GeneratorInterface
      *
      * @return string Result of generation
      */
-    public function generate($length = null)
```

Comment:
This change introduces issues. Both RandomGenerator and UniqIdGenerator implement GeneratorInterface, and that interface cannot be changed.
There is a reason for that null here :)